deployment/network-operator/README.md

```diff
@@ -216,6 +216,20 @@ Specifies components to deploy in order to facilitate a secondary network in Kub
 | `ipamPlugin.repository` | string | `dougbtv` | IPAM CNI Plugin image repository  |
 | `ipamPlugin.version` | string | `v0.3` | IPAM CNI Plugin image version  |
 
+## Helm Tests
+
+Network Operator has helm tests to verify deployment, currently supported tests:
+- Device Plugin Resource: This test create a pod that requests the first resource in `devicePlugin.resources`
+- RDMA Traffic: This test creates a pod that run server and client with `rping`
+
+Run the helm test with following command after deploying network operator with helm
+```
+$ helm test -n network-operator network-operator
+```
+Notes:
+- Test will keeping running endlessly if pod creating failed so it is recommended to use `--timeout` which fails test after exceeding given timeout
```

Comment:
>  Test will keeping running endlessly if pod creating failed so it is recommended to use `--timeout` which fails test after exceeding given timeout

so why not put --timeout in the command above ?

Reply:
added